storage.go

```diff
@@ -20,6 +20,10 @@ type Storage interface {
 	// the provided domain does not have an `Account` saved in the storage
 	// `ErrDomainNotFound` will be returned
 	Fetch(string) (Account, error)
+	// FetchAll retrieves all the `Account` objects from the fileStorage and
```

Comment:
small nit: the `Storage` interface is more generic than the `fileStorage` impl so I think this comment should stay generic as well.

```suggestion
	// FetchAll retrieves all the `Account` objects from the storage and
```